Documentation/gettingstarted/cni-chaining-kube-ovn.rst

```diff
@@ -0,0 +1,105 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+********
+Kube-OVN
+********
+
+This guide instructs how to install Cilium in chaining configuration on top of
+`Kube-OVN <https://github.com/kubeovn/kube-ovn>`_.
+
+.. include:: cni-chaining-limitations.rst
+
+Create a CNI configuration
+==========================
+
+Create a ``chaining.yaml`` file based on the following template to specify the
+desired CNI chaining configuration:
+
+
+.. code-block:: yaml
+
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: cni-configuration
+      namespace: kube-system
+    data:
+      cni-config: |-
+        {
+          "name": "generic-veth",
+          "cniVersion": "0.3.1",
+          "plugins": [
+            {
+              "type": "kube-ovn",
+              "server_socket": "/run/openvswitch/kube-ovn-daemon.sock",
+              "ipam": {
+                  "type": "kube-ovn",
+                  "server_socket": "/run/openvswitch/kube-ovn-daemon.sock"
+              }
+            },
+            {
+              "type": "portmap",
+              "snat": true,
+              "capabilities": {"portMappings": true}
+            },
+            {
+              "type": "cilium-cni"
+            }
+          ]
+        }
+
+Deploy the :term:`ConfigMap`:
+
+.. code-block:: shell-session
+
+   kubectl apply -f chaining.yaml
+
+
+Change Kube-OVN default install options
+=======================================
+
+To enable Cilium chaining mode and related functions, when install Kube-OVN the
+overlapped security function should be disabled and the cni conf priority should
+be modified.
```

Comment:
Would be nice to have a sentence describing why this changes are necessary

Reply:
add description why these options are needed